Cannot find name 'module'. in Web + mobile code sharing new collection

Linux Ubuntu 18.04 LTS
NPM version: 6.2.0
node version: 8.11.3

## Tell us about the problem

I used `ng new -c=@nativescript/schematics --name=simulator --shared`
I tried to run `ng serve` on my Angular + NativeScript project. and then I got:

`ERROR in src/app/app.component.ts(5,13): error TS2304: Cannot find name 'module'.
src/app/barcelona/player-detail/player-detail.component.ts(9,13): error TS2304: Cannot find name 'module'.
src/app/barcelona/players/players.component.ts(8,13): error TS2304: Cannot find name 'module'.
`
## Which platform(s) does your issue occur on?
web

## Please provide the following version numbers

### package.json

{
  "name": "simulator",
  "nativescript": {
    "id": "org.nativescript.simulator"
  },
  "version": "0.0.0",
  "scripts": {
    "ng": "ng",
    "start": "ng serve",
    "build": "ng build",
    "test": "ng test",
    "lint": "ng lint",
    "e2e": "ng e2e"
  },
  "private": true,
  "dependencies": {
    "@angular/animations": "^6.0.3",
    "@angular/common": "^6.0.3",
    "@angular/compiler": "^6.0.3",
    "@angular/core": "^6.0.3",
    "@angular/forms": "^6.0.3",
    "@angular/http": "^6.0.3",
    "@angular/platform-browser": "^6.0.3",
    "@angular/platform-browser-dynamic": "^6.0.3",
    "@angular/router": "^6.0.3",
    "core-js": "^2.5.4",
    "nativescript-angular": "^6.0.0",
    "nativescript-theme-core": "~1.0.4",
    "reflect-metadata": "~0.1.8",
    "rxjs": "^6.0.0",
    "tns-core-modules": "~4.0.0",
    "zone.js": "^0.8.26"
  },
  "devDependencies": {
    "@angular-devkit/build-angular": "0.7.0-beta.0",
    "@angular/cli": "^6.1.0-rc.2",
    "@angular/compiler-cli": "^6.0.3",
    "@angular/language-service": "^6.0.3",
    "@types/jasmine": "~2.8.6",
    "@types/jasminewd2": "~2.0.3",
    "@types/node": "~8.9.4",
    "codelyzer": "~4.2.1",
    "jasmine-core": "~2.99.1",
    "jasmine-spec-reporter": "~4.2.1",
    "karma": "~1.7.1",
    "karma-chrome-launcher": "~2.2.0",
    "karma-coverage-istanbul-reporter": "~2.0.0",
    "karma-jasmine": "~1.1.1",
    "karma-jasmine-html-reporter": "^0.2.2",
    "nativescript-dev-typescript": "~0.7.0",
    "nativescript-dev-webpack": "^0.14.0",
    "protractor": "~5.3.0",
    "ts-node": "~5.0.1",
    "tslint": "~5.9.1",
    "typescript": "2.7.2"
  }
}

## Please tell us how to recreate the issue in as much detail as possible

I created a new code sharing project following the webinar NS 4.1 steps

`npm i -g @angular/cli@6.1.0-beta.0` 
`npm i -g @nativescript/schematics`
`ng new -c@nativescript/schematics --name=simulator --shared`
`cd simulator/`
`npm i`
`ng serve -o`

I got this error:

`ERROR in src/app/app.component.ts(5,13): error TS2304: Cannot find name 'module'.
src/app/barcelona/player-detail/player-detail.component.ts(9,13): error TS2304: Cannot find name 'module'.
src/app/barcelona/players/players.component.ts(8,13): error TS2304: Cannot find name 'module'.
`

_**I have proved with either `@angular/cli@6.1.0-beta.0` or `@angular/cli@6.1.0-rc.2`**_ 

Any suggestions?

--- a/src/add-ns/_ns-files/__sourceDir__/app/__entryComponentName@dasherize__.component__nsext__.ts
+++ b/src/add-ns/_ns-files/__sourceDir__/app/__entryComponentName@dasherize__.component__nsext__.ts
@@ -2,7 +2,6 @@ import { Component } from '@angular/core';
 
 @Component({
   selector: '<%= indexAppRootTag %>',
-  moduleId: module.id,
   templateUrl: '<%= entryComponentImportPath %>.html',
 })
 

--- a/src/add-ns/_ns-files/__sourceDir__/app/barcelona/player-detail/player-detail.component.ts
+++ b/src/add-ns/_ns-files/__sourceDir__/app/barcelona/player-detail/player-detail.component.ts
@@ -6,7 +6,6 @@ import { PlayerService } from '../player.service';
 
 @Component({
   selector: 'ns-details',
-  moduleId: module.id,
   templateUrl: './player-detail.component.html',
 })
 export class PlayerDetailComponent implements OnInit {

--- a/src/add-ns/_ns-files/__sourceDir__/app/barcelona/players/players.component.ts
+++ b/src/add-ns/_ns-files/__sourceDir__/app/barcelona/players/players.component.ts
@@ -5,7 +5,6 @@ import { PlayerService } from '../player.service';
 
 @Component({
   selector: 'ns-players',
-  moduleId: module.id,
   templateUrl: './players.component.html',
 })
 export class PlayersComponent implements OnInit {

--- a/src/generate/component/index.ts
+++ b/src/generate/component/index.ts
@@ -77,7 +77,7 @@ export default function (options: ComponentOptions): Rule {
     },
 
     (tree: Tree) => {
-      if (platformUse.useNs) {
+      if (platformUse.nsOnly) {
         insertModuleId(tree, componentInfo.classPath);
       }
     },

--- a/src/migrate-component/index.ts
+++ b/src/migrate-component/index.ts
@@ -18,7 +18,6 @@ import { dirname, basename } from 'path';
 import { Schema as MigrateComponentSchema } from './schema';
 
 import { getSourceFile, addExtension, findRelativeImportPath } from '../utils';
-import { insertModuleId } from '../ast-utils';
 import { ComponentInfo, parseComponentInfo } from './component-info-utils';
 import { getNsConfigExtension, Extensions } from '../generate/utils';
 
@@ -37,8 +36,6 @@ export default function(options: MigrateComponentSchema): Rule {
     (tree: Tree, context: SchematicContext) => {
       componentInfo = parseComponentInfo(options)(tree, context);
     },
-    (tree: Tree) => 
-      updateComponentClass(tree, componentInfo),
     
     (tree: Tree, context: SchematicContext) =>
       addNsFiles(componentInfo, options)(tree, context),
@@ -47,9 +44,6 @@ export default function(options: MigrateComponentSchema): Rule {
       addComponentToNsModuleProviders(componentInfo, options)(tree)
   ]);
 }
-
-const updateComponentClass = (tree: Tree, componentInfo: ComponentInfo) => 
-  insertModuleId(tree, componentInfo.componentPath)
 
 const addNsFiles = (componentInfo: ComponentInfo, options: MigrateComponentSchema) => (tree: Tree, context: SchematicContext) => {
   context.logger.info('Adding {N} files');

--- a/src/ng-new/shared/_files/__sourcedir__/app/app.component.ts
+++ b/src/ng-new/shared/_files/__sourcedir__/app/app.component.ts
@@ -2,7 +2,6 @@ import { Component } from '@angular/core';
 
 @Component({
   selector: '<%= prefix %>-root',
-  moduleId: module.id,
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.<%= style %>']
 })

--- a/src/ng-new/shared/_files/__sourcedir__/app/barcelona/player-detail/player-detail.component.ts
+++ b/src/ng-new/shared/_files/__sourcedir__/app/barcelona/player-detail/player-detail.component.ts
@@ -6,7 +6,6 @@ import { PlayerService } from '../player.service';
 
 @Component({
   selector: '<%= prefix %>-details',
-  moduleId: module.id,
   templateUrl: './player-detail.component.html',
 })
 export class PlayerDetailComponent implements OnInit {

--- a/src/ng-new/shared/_files/__sourcedir__/app/barcelona/players/players.component.ts
+++ b/src/ng-new/shared/_files/__sourcedir__/app/barcelona/players/players.component.ts
@@ -5,7 +5,6 @@ import { PlayerService } from '../player.service';
 
 @Component({
   selector: '<%= prefix %>-players',
-  moduleId: module.id,
   templateUrl: './players.component.html',
 })
 export class PlayersComponent implements OnInit {

